static/loading-bar.js

```diff
@@ -10,18 +12,27 @@ function LoadingBar(color, parent) {
   return this;
 }
 
-LoadingBar.prototype.start = function(percentage) {
-  percentage = percentage || 0;
-  this.meter.css('width', percentage + '%');
+LoadingBar.prototype.start = function(progress) {
+  if(this.timer){
+    clearTimeout(this.timer);
+    this.timer = null;
+  }
+  this.meter.css('width', progress || starting + '%');
   this.elem.show();
 };
 
 LoadingBar.prototype.end = function() {
-  this.elem.hide();
+  this.update(100);
 };
 
-LoadingBar.prototype.update = function(percentage) {
-  this.meter.css('width', percentage + '%');
+LoadingBar.prototype.update = function(progress) {
+  this.meter.css('width', progress + '%');
+  if(progress === 100){
+    this.timer = setTimeout(function(){
+      console.log('hide');
```

Comment:
👋

Reply:
*face palm* I need a linter rule for `console.log` 😝